### PR TITLE
Proof-of-concept - patch mode

### PR DESCRIPTION
**Pull request by JonShort**
This is to prove that a mode where we only update the patch version is possible - #7 

@jaysc - you mind helping with some testing on this one?

### CODE REVIEW

File: src/npm_cmd.rs [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+}
+
+#[cfg(test)]
+mod patch_mode_init {
+    use super::*;
+    use serial_test::serial;
+    use std::{env, path::Path};
+
+    #[test]
+    #[serial]
```
Comment (jaysc) [inline src/npm_cmd.rs:296]: needs to run `serial` as this and the test below both interact with the same file

**Comment by jaysc** [comment]
I just realised you ment testing on windows, not adding unit tests 🤦 sorry

**Comment by jaysc** [comment]
Tested on Windows and Linux. Both works fine.

image

**Review by jaysc — DISMISSED** [review]

**Comment by jaysc** [comment]
Update readme

**Comment by JonShort** [comment]
> Update readme

@jaysc Done in [35b34d9](https://github.com/JonShort/npm-bumpall/pull/12/commits/35b34d98045d152c283a69106a98f34dcc56e885)

**Review by jaysc — APPROVED** [review]

**Pull request merged**